server_addon/deadline/client/ayon_deadline/plugins/publish/submit_aftereffects_deadline.py

```diff
@@ -57,7 +57,7 @@ def get_job_info(self):
         dln_job_info.Plugin = "AfterEffects"
         dln_job_info.UserName = context.data.get(
             "deadlineUser", getpass.getuser())
-        if self._instance.data["frameEnd"] > self._instance.data["frameStart"]:
+        if self._instance.data["frameEnd"] >= self._instance.data["frameStart"]:  # noqa
```

Comment:
Wait, so it must be filled even if you render single frame? In that case why is this condition here?

Reply:
Probably protection to flipped frameStart and frameEnd, but in that case it would failed later, so this is not necessary.